package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-native-iap",
   "version": "7.3.0",
   "description": "React Native In App Purchase Module.",
-  "main": "index.js",
+  "main": "index",
```

Comment:
Can you tell exact effect of changing this?

Reply:
Hi @hyochan , not sure but when adding this lib from my fork. I can not find `index.js` from `node_modules/react-native-iap`. So I remove `.js` so that it can pick `index.ts` instead

Reply:
@r0b0t3d Could you kindly test out after reverting this? In every `npm` tutorial, they recommend using `.js` extension. This update should be merged after we are sure with the actual consequences.

Reply:
Actually, I think you are using your own `ts` file instead of built one. you may try the `built` package using `yarn build` or `npm pack` and rename that to `react-native-iap` and move into `Example/node_modules`.

I think this would help!

Reply:
I'll try then let you know. Thanks